projects/canopy/src/lib/forms/radio/radio-button.component.scss

```diff
@@ -83,3 +93,15 @@
     background-color: var(--form-error-border-color);
   }
 }
+
+.lg-radio-button__label--sm::after {
+  --_radio-inner-margin-horizontal: calc(var(--radio-inner-height) / 2);
```

Comment:
The approach of using the css variables like this and marking them as `private` is taken from a video I've watched during lightning learning last week: https://www.youtube.com/watch?v=ZuZizqDF4q8&t=2146s

Reply:
An interesting, sounds good. I was going to say that we should add this to our styling best practices documentation, but then I released that we don't have one - I think? Maybe we could benefit from a markdown file explaining how we handle SCSS, e.g. using BEM, use of CSS custom properties, etc - and also private CSS vars like this too. Bu that's just a side note really.

Reply:
That's a great idea 👍 I will make sure to create a ticket for it in our backlog.